chapter_multilayer-perceptrons/underfit-overfit.md

```diff
@@ -623,9 +623,11 @@ the complex model overfits the data.
 
 ```{.python .input}
 #@tab all
+# Suppose only the first 50 train data are available for training
+n_train_available = 50
```

Comment:
it'll be better if the only change here is "Higher-Order Polynomial Function", otherwise hard to tell which factor contributes to overfitting

Reply:
for example, if you'd like to make n smaller, n has to be the same in other places. So the only difference in this analysis is the model, not the size of data